strawberry_django/auth/mutations.py

```diff
@@ -0,0 +1,26 @@
+import strawberry
+from django.contrib import auth
+from ..resolvers import django_resolver
+
+def login():
+    @strawberry.mutation
+    @django_resolver
+    def login(info, username: str, password: str):
+        request = info.context.request
+        user = auth.authenticate(request, username=username, password=password)
+        if user is not None:
+            auth.login(request, user)
+            return user
+        auth.logout(request)
+        return request.user
+    return login
+
+def logout():
```

Comment:
I don't see how string return type makes sense here, although I do not have much better suggestions, but maybe a bool, and `return True`?

Reply:
I have the same feeling. Boolean is better. Changed